src/calculateRentalCost.js

```diff
@@ -4,7 +4,28 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const oneDayCost = 40;
+  const shortTerm = 3;
+  const shortTermDiscount = 20;
+  const longTerm = 7;
+  const longTermDiscount = 50;
+  let basePrice = 0;
+
+  for (let i = 0; i < days; i++) {
+    basePrice += oneDayCost;
+  }
+
+  switch (true) {
+    case days >= longTerm:
+      basePrice -= longTermDiscount;
+      break;
+
+    case days >= shortTerm:
+      basePrice -= shortTermDiscount;
+      break;
+  }
```

Comment:
The `switch` statement is less clear than using an `if` statement in this case. It's better to use an `if` statement with `return` to simplify the code and make it more readable. Also, you can remove the `else` after `if` with `return` as it is not needed.